src/backend/TrafficCourts/Common/Features/FilePersistence/MinioFilePersistenceService.cs

```diff
@@ -243,7 +243,8 @@ public override async Task<string> SaveJsonFileAsync<T>(T data, string filename,
         catch (ObjectNotFoundException exception)
         {
             _logger.LogInformation(exception, "Object not found");
-            throw new FileNotFoundException("File not found", filename, exception);
+            if (_bucketName != "traffic-ticket-dev") throw new FileNotFoundException("File not found", filename, exception); // allow this in dev and docker-compose
```

Comment:
i will do the app.Environment.IsDevelopment() thingy that sounds like the best choice.  This particular error will occur in both the docker compose build and in dev since they both use the same oracle database but can use different object storage. E.g. if you upload a handwritten ticket through your docker-compose environment the db records will be the dev oracle database, but there will be no ticket image or JSON file in dev.

Reply:
Ah, okay, very good.